build_tools/cmake/iree_lit_test.cmake

```diff
@@ -84,6 +84,7 @@ function(iree_lit_test)
     "TEST_TMPDIR=${IREE_BINARY_DIR}/tmp/${_NAME}_test_tmpdir"
     "LIT_OPTS=-v"
     "FILECHECK_OPTS=--enable-var-scope")
+  set_property(TEST ${_NAME_PATH} PROPERTY TIMEOUT ${_RULE_TIMEOUT})
```

Comment:
Just curious - what is the output like if a timeout is exceeded? (It's also useful to check that this actually works :P)
Can you try with a timeout of 1 second to see what happens?

Reply:
Instead of `(Passed)` or `(Failed)` we get `(Timeout Exceeded)` in CTest. I tested with a timeout of `1` before sending out the PR.